### DISCUSSION TITLE
Feature support

### DISCUSSION THREAD

**Discussion by Dretreyt** [Budibase/budibase#14576] [Help]
**Describe the feature request**
A clear and concise description of what the feature request

In budibase how would I create the following concept.  

I need to create a school reporting type system where a referral can be added to a database into a table etc... and include an email address for example and to auto generate a pin for this record, this I have already. 

I then want the step in the process will be the manual sending of the next form URL to the parent of the student. 

But before they can access the record and add data etc... they will need to provide the email address and pin number associated to that 1 record before they can view it. Or I other words, they provide their email address and the pin and the record both are associated get loaded into a form.

So it will be a public form page but they will need to provide data to view the form and provide data etc.

**Comment by ConorWebb96** [comment]
Hey @Dretreyt,

This sounds very doable with Budibase. I think you should read the documentation. Or check out the university videos to get started!
